web/client/codechecker_client/cmd/store.py

```diff
@@ -386,28 +548,62 @@ def collect_file_hashes_from_plist(plist_file):
                     "again to update the reports!", changed_files)
         sys.exit(1)
 
-    with zipfile.ZipFile(zip_file, 'a', allowZip64=True) as zipf:
+    hash_to_file = {}
+    # There can be files with same hash,
+    # but different path.
+    file_to_hash = {}
+
+    # all the mentioned source files in the reports
+    for source_file, info in source_file_info.items():
+        if bool(info):
+            file_to_hash[source_file] = info['hash']
+            hash_to_file[info['hash']] = source_file
+        else:
+            # source file is missing
+            pass
```

Comment:
This else branch does nothing. What is the main purpose of it? Couldn't we remove it?